src/lib/utils/utils.ts

```diff
@@ -179,7 +178,7 @@ type FormatCurrencyProps = {
  */
 export const formatCurrency = (
   amount: BigSource,
-  { currency = 'USD', showCurrency = false }: FormatCurrencyProps = {},
+  { currency = 'USD', showCurrency = false, showCurrencySymbol = true }: FormatCurrencyProps = {},
```

Comment:
I think that this should be false by default

Reply:
if it was then it would change the behaviour of the current use-cases of this function that do not rely on this parameter.